wordpress-mobile/gutenberg-mobile
# Updates Android Gradle Plugin to 4.0.2

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/32144

To test:
Run an Android composite build with `gutenberg-mobile`:
1. Open the `WordPress-Android` project
2. Set your `gutenberg-mobile` path (`localGutenbergMobilePath`) at `local-builds.gradle`
3. Reload Gradle project and run
4. **Verify** that no Gradle error occurs due to version mismatch

PR submission checklist:
